crates/bins/cli-wallet/src/main.rs

```diff
@@ -604,11 +602,68 @@ async fn main() -> Result<()> {
                 match wad.value() {
                     Ok(v) => println!("Total Value: {} {}", v, wad.unit()),
                     Err(_) => {
-                        println!("sum of all proofs in the wad overflowed");
+                        println!("The sum of all proofs in the wad overflowed.");
                         continue;
                     }
                 };
+
+                if verify {
```

Comment:
move the content of this block in a `cli-wallet/src/delq.rs` file to avoid overcrowding the `main.rs` file